### PR TITLE
2064-Config-Selector-Dropdown---Add-tags

### LINKED ISSUE #2064: Config Selector Dropdown - Add tags

### PR DESCRIPTION
**Pull request by mainawycliffe**
Fixes #2064

### CODE REVIEW

File: src/components/Forms/Formik/FormikResourceSelectorDropdown.tsx [hunk 1]
```diff
@@ -48,49 +50,56 @@ export default function FormikResourceSelectorDropdown({
...
+        return data.configs.map((config) => {
+          const tags = Object.values(config.tags ?? {}).map((value) => value);
+
+          return {
+            icon: <ConfigIcon config={config} />,
+            value: config.id,
+            label: (
+              <div className="flex flex-wrap gap-1">
+                <span className="mr-2"> {config.name}</span>
+                <Tag title="tags" variant="blue">
```
Comment (moshloop) [inline src/components/Forms/Formik/FormikResourceSelectorDropdown.tsx:91]:
```suggestion
                <Tag title="tags">
```

File: src/components/Forms/Formik/FormikResourceSelectorDropdown.tsx [hunk 2]
```diff
@@ -48,49 +50,56 @@ export default function FormikResourceSelectorDropdown({
...
+                  {config.type.split("::").at(-1)?.toLocaleLowerCase()}
+                </Tag>
+                {tags.map((tag) => (
+                  <Tag title={tag} key={tag} variant="blue">
```
Comment (moshloop) [inline src/components/Forms/Formik/FormikResourceSelectorDropdown.tsx:95]:
```suggestion
                  <Tag title={tag} key={tag}>
```

**Comment by moshloop** [comment]
@mainawycliffe search is still broken - and the tags do not have any styling now.

**Comment by mainawycliffe** [comment]
Working on a fix for search

**Pull request merged**